src/sentry_plugins/asana/plugin.py

```diff
@@ -24,6 +25,21 @@ class AsanaPlugin(CorePluginMixin, IssuePlugin2):
     conf_key = "asana"
     auth_provider = "asana"
     required_field = "workspace"
+    feature_descriptions = [
+        FeatureDescription(
+            """
+            Create and link Sentry issue groups directly to an Asana ticket in any of your
+            projects, providing a quick way to jump from a Sentry bug to tracked ticket!
+            """,
+            IntegrationFeatures.ISSUE_BASIC,
+        ),
+        FeatureDescription(
+            """
+            Link Sentry issues to existing Asana tickets.
+            """,
+            IntegrationFeatures.ISSUE_BASIC,
+        ),
```

Comment:
@scefali Should we move feature descriptions into one object in a constants file, rather than hard code descriptions in each plugin folder?

Reply:
@NisanthanNanthakumar you mean the actual description text? Those descriptions are actually specific to a plugin/integration so they belong where they currently sit.

Reply:
@scefali yes. constant strings could be hoisted into a constant file that can be imported and references as `descriptions[asana]`. This way we would have all the descriptions/permissions for each plugin in one place.

Reply:
@NisanthanNanthakumar I would prefer to keep the convention we already have with integrations where they are defined in the file specific to that integration.

Reply:
Also, I'm not a fan of making constants if the value is only going to be used one time at most